prac_05/hex_colours.py

```diff
@@ -0,0 +1,14 @@
+"""
+CP1404/CP5632 Practical
+Colours in a dictionary
+"""
+COLOURS = {'Blanched Almond': '#ffebcd', 'Bright Turquoise': '#08e8de', 'Dodger Blue1': '#1e90ff', 'Falu Red': '#801818',
```

Comment:
Could the name be changed to clearly show it's a dictionary? Maybe COLOURS_TO_HEX?